StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -23,6 +23,7 @@
     /// delegates, enums, events, finalizers, indexers, interfaces, methods, properties, and structs.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: I believe this one can be generated, at least as an element containing no text. The result would then produce several other warnings related to the specific generated empty elements, but at least the user wouldn't have to type the XML portion manually.

A new issue would be necessary to discuss the details of this algorithm, since it varies by element.